lib/rhc/commands/deployment.rb

```diff
@@ -74,7 +74,9 @@ def activate(id)
 
       raise RHC::DeploymentsNotSupportedException.new if !rest_app.supports? "DEPLOY"
 
-      activate_deployment(rest_app, id)
+      ssh_executable = check_ssh_executable! options.ssh
+
+      activate_deployment(rest_app, id, ssh_executable, !!options.ssh)
```

Comment:
I think it would be cleaner to define `activate_deployment` with a single parameter instead of both `ssh_executable` and the `use_custom_ssh` Boolean and just pass in the value of `options.ssh && check_ssh_executable! options.ssh` as the argument here.  Is there a reason not to take that approach?